test/where_test.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+import {where} from '../src/where.js';
+import {Tensor} from '../src/lib/tensor.js';
+import * as utils from './utils.js';
+
+describe('test where', function() {
+  function testWhere(condition, inputA, inputB, expected) {
+    const tensorCondition = new Tensor(condition.shape, condition.data);
+    const tensorA = new Tensor(inputA.shape, inputA.data);
+    const tensorB = new Tensor(inputB.shape, inputB.data);
+    const outputTensor = where(tensorCondition, tensorA, tensorB);
+    utils.checkShape(outputTensor, expected.shape);
+    utils.checkValue(outputTensor, expected.data);
+  }
+
+  it('where', function() {
+    const condition = {
+      shape: [2, 3],
+      data: [
+        1, 1, 0,
+        0, 1, 0,
```

Comment:
Please add at least one test case to cover !0=true condition data.

Reply:
Is it like this？
const condition = {
      shape: [2, 3],
      data: [
        2, 3, 0,
        0, 5, 0,
      ],
    };

Reply:
Yes, here input data is of Uint8, the value range is [0, 255], value 0 means selecting data from falseValues tensor, and others values mean selecting data from trueValues tensor.